ariac-competitor/build_competitor_image.bash

```diff
@@ -2,8 +2,22 @@
 
 DOCKER_ARGS=""
 # Uncoment this line to rebuild without cache
+# TODO: expose this as an argument
```

Comment:
it doesn't show up well in the diffs, sorry: [the no-cache option](https://github.com/osrf/ariac-docker/pull/6/files/1d787ab18d87b9530456c46ac696ee2fc90d3fc2#diff-a3996c3db93ff76671e5a51b86508a1cR6)